src/back/spv/writer.rs

```diff
@@ -10,6 +10,18 @@ enum Signedness {
     Signed = 1,
 }
 
+#[derive(Debug)]
+enum LocalType {
```

Comment:
we probably want vectors as well here, since it's implicitly required by the matrix types in our IR

Reply:
Good thing you bring this up. Today I have extensively tested the matrix capabilities, but the SPIR-V output is totally not correct yet. I am already working on fixing this, and adding the Vector `LocalType`.